var/spack/repos/builtin/packages/raja/package.py

```diff
@@ -62,12 +63,14 @@ class Raja(CachedCMakePackage, CudaPackage, ROCmPackage):
     depends_on("camp@0.2.2:0.2.3", when="@0.14.0")
     depends_on("camp@0.1.0", when="@0.10.0:0.13.0")
     depends_on("camp@2022.03.0:", when="@2022.03.0:")
+    depends_on("camp+openmp", when="+openmp")
```

Comment:
@trws - Umpire needs this line too, I think that will fix the problem.

Reply:
Sadly, that did not take care of it.  I think we may need to get the updates in a release to get past this. =/

Reply:
:( - will constraining the camp version help?

Reply:
I can try it, I'll add the new version bump I put up for camp and see if it's enough, or if we need to bump the others.

Reply:
It looks like that might actually do it, I bumped camp to 2022.03.2 on the theory we might have to bump the other two to bump camp (the submodule is still wrong) but bumping it in spack seems to make gitlab happy: https://gitlab.spack.io/spack/spack/-/pipelines/164770